### PR TITLE
Add Snap scripts to codebase

### PR DESCRIPTION
**Pull request by buggtb**
Here's the snap generation files. We'll need to to discuss on the mailing list how best to package these and any extra hooks I'm missing, but better to have them and improve them than to wait until its perfect.

### CODE REVIEW

File: tika-deployment/tika-snap-app/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: tika-app
+version: '1.16-SNAPSHOT'
+summary: Tika Server for metadata discovery and extraction
```
Comment (grossws) [inline tika-deployment/tika-snap-app/snapcraft.yaml]: Shouldn't that be `Tika App for metadata discovery and extraction`?

Comment (buggtb) [inline tika-deployment/tika-snap-app/snapcraft.yaml]: Indeed @grossws, fixed now.

**Comment by chrismattmann** [comment]
LGTM!

**Comment by chrismattmann** [comment]
Committing thanks @buggtb and @grossws for the review!

**Pull request merged**